browser/help/topics/v4-browser-hts.md

```diff
@@ -0,0 +1,223 @@
+      - `n_larger`: Number of individuals with a higher age than the highest bin.
+    - `hom`: Struct containing age distribution information for individuals homozygous for this variant. Structured to allow easy histogram creation. Contains same fields as `het` above.
+  - `filters`: Set containing variant QC filters.
+  - `quality_metrics`: Struct containing variant quality metric histograms information.
+    - `allele_balance`: Struct containing variant allele balance histograms information.
+      - `alt_adj`: Struct containing variant allele balance information calculated across high-quality genotypes. Structured to allow easy histogram creation. Contains same fields as other histogram structs (e.g., `het`). This data is displayed in the "Allele balance for heterozygotes" histogram in the browser's variant page.
+      - `alt_raw`: Struct containing variant allele balance information calculated across unadjusted genotypes. Structured to allow easy histogram creation. Contains same fields as other histogram structs (e.g., `het`).
+    - `genotype_depth`: Struct containing information used to display genotype depth (DP) histograms.
+      - `all_adj`: Struct containing DP information calculated using high-quality genotypes. Structured to allow easy histogram creation. Contains same fields as other histogram structs (e.g., `het`).
+      - `all_raw`: Struct containing DP information calculated across unadjusted genotypes. Structured to allow easy histogram creation. Contains same fields as other histogram structs (e.g., `het`).
+      - `alt_adj`: Struct containing DP information calculated using high-quality genotypes (variant carriers only). Structured to allow easy histogram creation. Contains same fields as other histogram structs (e.g., `het`).
+      - `alt_raw`: Struct containing DP information calculated across unadjusted genotypes (variant carriers only). Structured to allow ease of creating histograms. Contains same fields as other histogram structs (e.g. `het`)
+    - `genotype_quality`: Struct containing information used to display genotype quality (GQ) histograms.
+      - `all_adj`: Struct containing GQ information calculated using high-quality genotypes. Structured to allow easy histogram creation. Contains same fields as other histogram structs (e.g., `het`).
+      - `all_raw`: Struct containing GQ information calculated across unadjusted genotypes. Structured to allow easy histogram creation. Contains same fields as other histogram structs (e.g., `het`).
+      - `alt_adj`: Struct containing GQ information calculated using high-quality genotypes (variant carriers only). Structured to allow easy histogram creation. Contains same fields as other histogram structs (e.g., `het`).
+      - `alt_raw`: Struct containing GQ information calculated across unadjusted genotypes (variant carriers only). Structured to allow easy histogram creation. Contains same fields as other histogram structs (e.g., `het`).
```

Comment:
these descriptions are informative but a bit wordy (apologies if I suggested this previously). how about replacing
> Structured to allow easy histogram creation. Contains same fields as other histogram structs (e.g., `het`).

in each with just
>  Contains same fields as other histogram structs.